src/runtime-rs/crates/resource/src/share_fs/share_virtio_fs.rs

```diff
@@ -89,9 +89,16 @@ pub(crate) async fn setup_inline_virtiofs(id: &str, h: &dyn Hypervisor) -> Resul
             prefetch_list_path: None,
         },
     };
-    h.add_device(DeviceType::ShareFsMount(virtio_fs))
+
+    let result = h
+        .add_device(DeviceType::ShareFsMount(virtio_fs))
         .await
-        .with_context(|| format!("fail to attach passthrough fs {:?}", source))
+        .with_context(|| format!("fail to attach passthrough fs {:?}", source));
+
+    match result {
+        Ok(__) => Ok(()),
```

Comment:
Nit:

```suggestion
        Ok(_) => Ok(()),
```

? :smile:

Reply:
fixed :) It still compiled with that!